scripts/urlgrab.pl

```diff
@@ -55,11 +56,12 @@ sub find_url {
 }
 
 sub url_log{
+   my $t = localtime;
    my($where,$channel,$url) = @_;
    return if lc $url eq lc $lasturl; # a tiny bit of protection from spam/flood
    $lasturl = $url;
    open(URLLOG, ">>", $file) or return;
-   print URLLOG time." $where $channel $lasturl\n";
+   print URLLOG $t->datetime . " $channel $where $lasturl\n";
```

Comment:
I guess I'm probably the only person who has anything to read this format, easy enough to deal with datetime changing and probably better.

Could we not swap `$channel` and `$where` randomly though, please.